scoped/authentication/authentication-indirect.md

```diff
@@ -0,0 +1,38 @@
+**Name**  
+Authentication (indirect)
+
+----
+**Actor**  
+Anonymous user
+
+----
+**Subject area**  
+Individual question page
+
+----
+**Precondition**  
+User wants to answer question
+
+----
+**Termination outcome**  
+User is signed in and back on the original page
+
+----
+**Basic flow**
+
+- User sees "sign in to answer this question" link/button and clicks.
```

Comment:
I guess for consistency we should say "log in or sign up", not "sign in" which is probably my typo.